UI/Controllers/MentorController.cs

```diff
@@ -17,52 +17,55 @@ public class MentorController : ControllerBase
     {
         private readonly ICandidateService _candidateService;
         private readonly IEmployeeService _employeeService;
-        private readonly IInternshipTeamService _internshipTeam;
+        private readonly IInternshipTeamService _internshipTeamService;
+        private readonly ISpecializationService _specializationService;
+        private readonly IEnglishLevelService _englishLevelService;
+        private readonly ICityService _cityService;
+        private readonly IMapper _mapper;
         private readonly UserManager<User> _userManager;
 
-        public MentorController(ICandidateService candidateService, IInternshipTeamService internshipTeam, IEmployeeService employeeService, UserManager<User> userManager)
+        public MentorController(ICandidateService candidateService,
+                                ISpecializationService specializationService,
+                                IInternshipTeamService internshipTeamService,
+                                IEmployeeService employeeService,
+                                IEnglishLevelService englishLevelService,
+                                ICityService cityService,
+                                IMapper mapper,
+                                UserManager<User> userManager)
         {
             _candidateService = candidateService;
             _userManager = userManager;
             _employeeService = employeeService;
-            _internshipTeam = internshipTeam;
+            _internshipTeamService = internshipTeamService;
+            _specializationService = specializationService;
+            _englishLevelService = englishLevelService;
+            _cityService = cityService;
+            _mapper = mapper;
         }
 
-        // GET: api/<MentorController>
-        [HttpGet]
+        [HttpGet("candidates")]
         [Authorize(Roles = "admin, mentor")]
         public async Task<List<CandidateDTO>> GetCandidates()
         {
             var user = await _userManager.FindByNameAsync(User.Identity.Name);
             var employee = _employeeService.GetEmployeeByEmail(user.Email);
-            var currentTeam = _internshipTeam.GetInternshipTeamByEmployeeId(employee.Id);
-            var result = _candidateService.GetCandidatesFromTeam(currentTeam.TeamNumber);
-            return result.ToList();
+            var currentTeam = _internshipTeamService.GetInternshipTeamByEmployeeId(employee.Id);
+            var candidates = _candidateService.GetCandidatesFromTeam(currentTeam.TeamNumber);
+            return candidates.ToList();
         }
 
-        // GET api/<MentorController>/5
-        [HttpGet("{id}")]
-        public string Get(int id)
-        {
-            return "value";
-        }
-
-        // POST api/<MentorController>
-        [HttpPost]
-        public void Post([FromBody] string value)
+        [HttpGet("{id}/form")]
+        [Authorize(Roles = "admin, mentor")]
+        public CandidateForMentorDTO GetForm(int id)
         {
```

Comment:
why you do not use CandidateDTO?
do you realy need to use string representation of city, specialization and english level?